test_pytest_html.py

```diff
@@ -37,6 +48,7 @@ def assert_summary(html, tests=1, duration=None, passed=1, skipped=0, failed=0,
     assert int(re.search('(\d)+ errors', html).group(1)) == errors
     assert int(re.search('(\d)+ expected failures', html).group(1)) == xfailed
     assert int(re.search('(\d)+ unexpected passes', html).group(1)) == xpassed
+    assert_summary_checkboxes(html, passed)
```

Comment:
I think it may make sense to also add a test that opens the HTML report in a browser and verifies that toggling the checkboxes filters the results. This would have caught the issue I'm seeing with filtering errors.